pkg/api/api.go

```diff
@@ -35,6 +39,7 @@ func NewScratchDataAPI(storageServices *models.StorageServices, destinationManag
 		destinationManager: destinationManager,
 		dataSink:           dataSink,
 		snow:               snow,
+		cache:              storageServices.Cache,
```

Comment:
This is not needed. Any functions that need the cache can just call `storageService.Cache`